cumulusci/cli/cci.py

```diff
@@ -753,8 +754,6 @@ def service_info(config, service_name, plain):
 
 
 # Commands for group: org
-
-
```

Comment:
The blank lines here were intentional since the comment is describing the group of several commands that follows; not just org_browser.